### ISSUE TITLE
excessive whitespace in preferences dialog after resizing

### ISSUE THREAD

**Issue by ghost** [bleachbit/bleachbit#1290]
![Снимок экрана от 2021-11-04 06-37-53](https://user-images.githubusercontent.com/91217152/140254661-def3bb6f-1761-4e38-bf56-df2af9a72e9f.png)
![Снимок экрана от 2021-11-04 06-37-37](https://user-images.githubusercontent.com/91217152/140254663-53a06f6a-7bde-433e-9a67-33cc1c14311f.png)
![Снимок экрана от 2021-11-04 06-37-10](https://user-images.githubusercontent.com/91217152/140254667-90b62142-fc5e-4779-ae96-82a39f7c7c82.png)

**Comment by abitrolly** [comment]
Which BleachBit version?

**Comment by monstrik1** [comment]
4.4.1

**Comment by abitrolly** [comment]
I can confirm the layout bug - after resizing settings window the internal content is not resized properly.

**Comment by az0** [comment]
Yes, I see that too on Ubuntu. I retitled it to: _excessive whitespace in preferences dialog after resizing_. This is a relatively low priority because it's not a functional problem, and it's easy to avoid. Still, it's worth taking a look later

**Comment by monstrik1** [comment]
[Errno 13] - What does it mean?

**Comment by abitrolly** [comment]
@monstrik1 can mean many things depending on where it appears.

**Comment by az0** [comment]
Errno normally means permission denied. This doesn't seem to relate to the whitespace issue. If you have another problem, please open another ticket.

**Comment by monstrik1** [comment]
> Errno обычно означает отказ в разрешении. Похоже, это не связано с пробелов. Если у вас возникла другая проблема, откройте другой тикет.

I just asked. something doesn't erase. there was this error.
